### ISSUE TITLE
[Bug Report][3.0.0] v-img alt text prop support is missing

### ISSUE THREAD

**Issue by TwinHits** [vuetifyjs/vuetify#16203]
### Environment
**Vuetify Version:** 3.0.0
**Vue Version:** 3.2.45
**Browsers:** Chrome 107.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
1. Put a alt prop in v-img with "test alt test"
2. Hover over v-img once rendered
3. Inspect <img tag in dev tools

### Expected Behavior
If hovering, should show the "test alt test" alt text.
If inspecting, should show alt="test alt test"

### Actual Behavior
No alt text appears when hovering
The alt attribute appears when inspecting, but it is empty.

### Reproduction Link
[https://codepen.io/TwinHits/pen/VwdErmv](https://codepen.io/TwinHits/pen/VwdErmv)

### Other comments
This is important for 508 compliance. This appears to be an issue with line https://github.com/vuetifyjs/vuetify/blob/next/packages/vuetify/src/components/VImg/VImg.tsx#L195

**Comment by TwinHits** [comment]
I searched for duplicate issues but did not find any :( let me know if I am blind.

**Comment by KaelWD** [comment]
`alt` sets `aria-label` which should behave the same in screenreaders. 

> If hovering, should show the "test alt test" alt text.

This doesn't happen with `<img>` either, only `title` shows text on hover.

**Comment by larsvdd** [comment]
Does it not impact SEO that no alt-tag is set on the img itself?

**Comment by johnleider** [comment]
@KaelWD Is there anything actionable here?

**Comment by marjev** [comment]
Unfortunately, @KaelWD `should` doesn't mean it will. 😸 
Please, don't use `aria-label` since it is a lazy way of trying to establish accessibility. See: https://www.w3.org/WAI/ARIA/apg/practices/read-me-first/

Would it be possible to add `alt` here instead of `aria-label` to ensure that some (more strict) SRs like JAWS and NVDA play well with this? Using a native attribute instead of `aria-label` would ensure future reliability.

**Comment by johnleider** [comment]
> Unfortunately, @KaelWD `should` doesn't mean it will. smile_cat Please, don't use `aria-label` since it is a lazy way of trying to establish accessibility. See: https://www.w3.org/WAI/ARIA/apg/practices/read-me-first/
> 
> Would it be possible to add `alt` here instead of `aria-label` to ensure that some (more strict) SRs like JAWS and NVDA play well with this? Using a native attribute instead of `aria-label` would ensure future reliability.

want to take a stab at the fix?

**Comment by marjev** [comment]
@johnleider sure, but since I'm super busy these days, I can just give a quick plan on how this can be fixed and I believe it should be straightforward for your team to take it from there, mostly by QAing that all your use-cases are still covered.

- go to https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VImg/VImg.tsx

```const img = (
        <img
          class={['v-img__img', containClasses.value]}
          src={ normalisedSrc.value.src }
          srcset={ normalisedSrc.value.srcset }
          alt={ props.alt }
          sizes={ props.sizes }
          ref={ image }
          onLoad={ onLoad }
          onError={ onError }
        />
      )
```

```
const __preloadImage = () => (
      <MaybeTransition transition={ props.transition }>
        { normalisedSrc.value.lazySrc && state.value !== 'loaded' && (
          <img
            class={['v-img__img', 'v-img__img--preload', containClasses.value]}
            src={ normalisedSrc.value.lazySrc }
            alt={ props.alt }
          />
        )}
      </MaybeTransition>
    )
```

This is where things get a bit confusing to me, so I would like to understand what was the reasoning behind putting aria and role on the div, but I believe by providing the image with alt text, it should work.
**Notice that I've removed aria-label and role attributes**

```
<VResponsive
        class={[
          'v-img',
          { 'v-img--booting': !isBooted.value },
        ]}
        style={{ width: convertToUnit(props.width === 'auto' ? naturalWidth.value : props.width) }}
        aspectRatio={ aspectRatio.value }
        v-intersect={[{
          handler: init,
          options: props.options,
        }, null, ['once']]}
      >{{
        additional: () => (
          <>
            <__image />
            <__preloadImage />
            <__gradient />
            <__placeholder />
            <__error />
          </>
        ),
        default: slots.default,
      }}</VResponsive>
    ))
```

**Comment by marjev** [comment]
@johnleider also, if you wanted to establish something like this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role

see the example on that page:
```
<div role="img" aria-label="Description of the overall image">
  <img src="graphic1.png" alt="alternative text" />
  <img src="graphic2.png" alt="in case the images don't load" />
</div>
```

**Comment by ffxsam** [comment]
Opened #16841. Took me a whole 10 minutes. 😛
